features/admin.userstores.v1/pages/user-stores.tsx

```diff
@@ -274,7 +274,14 @@ const UserStores: FunctionComponent<UserStoresPageInterface> = (
     return (
         <PageLayout
             action={
-                (isLoading || !(!searchQuery && filteredUserStores?.length <= 0))
+                (
+                    isLoading
+                    || !(
+                        !searchQuery
+                        && filteredUserStores?.length <= 0
+                        && disabledFeatures?.includes("userStores.type.remote")
```

Comment:
Let's use already defined feature identifiers in FEATURE_DICTIONARY

Reply:
Added with e0f582ed3a1a78dd8a29d989e075eaa94ef8a7d1.